armi/nuclearDataIO/tests/test_xsLibraries.py

```diff
@@ -546,4 +493,4 @@ def test_mergeAllXSLibFiles(self):
 
 
 # Remove the abstract class, so that it does not run (all tests would fail)
-del TestXSlibraryMerging
```

Comment:
now that you renamed it, isn't there's no reason to delete it down here anymore so you can remove these lines. The test discovery finds all classes that start with `Test` in their name. Hmm maybe not b/c other test classes in here don't start with `Test`...

Reply:
Unfortunately, I tried deleting this line, and I get test failures:

```bash
  File "C:\Users\jstilley\codes\repos\nala\framework\armi\nuclearDataIO\tests\test_xsLibraries.py", line 303, in setUp
    self.libAA = self.getReadFunc()(self.getLibAAPath())
  File "C:\Users\jstilley\codes\repos\nala\framework\armi\nuclearDataIO\tests\test_xsLibraries.py", line 312, in getReadFunc
    raise NotImplementedError()
NotImplementedError
```

I guess `pytest` finds all classes that subclass `unittest.TestCase`, it doesn't just go by the name of the class.

Which is strange, because I'm pretty sure it _does_ find unit test methods by any name starting with `test`.  That's a strange asymmetry.

Reply:
> pytest will automatically collect unittest.TestCase subclasses and their test methods in test_*.py or *_test.py files

https://docs.pytest.org/en/7.1.x/how-to/unittest.html

Reply:
ah ok thanks for the reminder. so its the module name not the class name. I stand corrected.